web/src/app/pages/create-survey/job-details/job-details.component.html

```diff
@@ -16,8 +16,13 @@
 
 <h1 class="title">Add a job</h1>
 <h2 class="description">
-  Data collectors will use this job to know what data should be collected and
-  where.
+  Jobs let you define what data should be collected about which places. For each job, 
```

Comment:
@amegantz Also pinged you on the Figma about this, I'll wait for your input before sending for review.